Formula/fop.rb

```diff
@@ -20,8 +14,13 @@ class Fop < Formula
   end
 
   def install
+    rm_rf Dir["fop/*.bat"] # Remove Windows files.
     libexec.install Dir["*"]
-    bin.write_exec_script libexec/"fop"
+
+    executable = libexec/"fop/fop"
+    executable.chmod 0555
```

Comment:
Homebrew fixes to [0555 I believe](https://github.com/Homebrew/brew/blob/c9c7f462d37500549127efba96c7a25e5c70de4a/Library/Homebrew/cleaner.rb#L102-L105)?